worker/revpi_factory_reset.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+SSH_HOST_RPI=$1
+DEVICE_TYPE=$2
+DEVICE_SERIAL=$3
+DEVICE_MAC=$4
+
+echo "$SSH_HOST_RPI"
+echo "$DEVICE_TYPE"
+echo "$DEVICE_SERIAL"
+echo "$DEVICE_MAC"
+
```

Comment:
Maybe it is not needed, but what do you think about a function to show how to use this script?
I mean something like that:

`
usage() {
    echo "Usage: $0 ...[-h ssh_host_rpi] [-t device_type] [-s device_serial] [-m device_mac]" 1>&2
    exit 1
}

while getopts ":h:t:s:m:" opt; do
    case "${opt}" in
        h) SSH_HOST_RPI="${OPTARG}" ;;
        t) DEVICE_TYPE="${OPTARG}" ;;
	s) DEVICE_SERIAL="${OPTARG}" ;;
	m) DEVICE_MAC="${OPTARG}" ;;
        *) usage ;;
    esac
done
`

Reply:
Will think about it. The script is so small, that it almost makes no sense to add something like a usage description and/or getopts and it should be removed or replaced by a python script once the ctrl hardware is ready.

Reply:
Ok.
It is ok like that. We can add that in the future.

Just please remove one new line at the end of file and it is ok for me.